tst/regression/CMakeLists.txt

```diff
@@ -61,22 +58,27 @@ list(APPEND TEST_ARGS "--driver ${PROJECT_BINARY_DIR}/example/advection/advectio
 --num_steps 5")
 
 if (ENABLE_HDF5)
-  setup_test( output_hdf5 "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
-  --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/output_hdf5/parthinput.advection \
-  --num_steps 4")
-  setup_test_coverage( output_hdf5 "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
-  --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/output_hdf5/parthinput.advection \
-  --num_steps 4")
-  setup_test_mpi( "${NUM_MPI_PROC_TESTING}" output_hdf5
-    "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
-  --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/output_hdf5/parthinput.advection \
-  --num_steps 4")
-  setup_test_mpi_coverage("${NUM_MPI_PROC_TESTING}" output_hdf5
-    "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
-  --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/output_hdf5/parthinput.advection \
-  --num_steps 4")
+  list(APPEND TEST_DIRS output_hdf5)
+  list(APPEND TEST_PROCS ${NUM_MPI_PROC_TESTING})
+  list(APPEND TEST_ARGS "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
+    --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/output_hdf5/parthinput.advection \
+    --num_steps 4")
+  # h5py is only neeed if hdf5 is enabled
+  list(APPEND REQUIRED_PYTHON_MODULES h5py) 
 endif()
 
+# Any modules that are required by python can be added to REQUIRED_PYTHON_MODULES list variable,
+# before including TestSetup.cmake. 
+list(APPEND REQUIRED_PYTHON_MODULES argparse errno math matplotlib numpy os shutil) 
+list(APPEND REQUIRED_PYTHON_MODULES subprocess sys) 
```

Comment:
You can just split the arguments to a function call in CMake across multiple lines.